README.md

```diff
@@ -41,6 +41,113 @@ $ cargo run
 
 The API documentation is available from the server by opening the URL http://127.0.0.1:8000 with the web browser.
 
+# Cross Compiling
+
+## Raspberry Pi OS (64-bit)
+
+### Already done
+
+We have already added the following to the `.cargo/config.toml`
+
+```toml
+[build]
+
+# Raspberry Pi 4 64-bit
+[target.aarch64-unknown-linux-gnu]
+linker = "aarch64-none-linux-gnu-gcc"
+```
+
+This allows cross-compiling for the 64-bit version of the Raspberry Pi OS. 
+
+### Install standard library
+
+Assuming you already have a Rust development environment for the host system. Installing a standard library for our target system is easy as
+
+```bash
+$ rustup target add aarch64-unknown-linux-gnu
+```
+
+### Download GNU Toolchain
+
+We need to install the correct version of the toolchain, which we can check by running the following command on Raspberry Pi:
+
+```bash
+$ gcc --version
+gcc (Debian 10.2.1-6) 10.2.1 20210110
+Copyright (C) 2020 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+```
+
```

Comment:
Using crossbuild-essential-arm64 should be enough for ubuntu/debian.

Reply:
Thanks for the tip. We tried this, and it works excellently after some tool name changes. We add instructions for Debian-based distributions to the README file.